test/bigtest/network/configs/interfaces.js

```diff
@@ -0,0 +1,18 @@
+import { noop } from 'lodash';
+import { INTERFACES_API } from '../../../../src/common/constants';
+
+const configInterfaces = server => {
+  server.get(INTERFACES_API, (schema) => {
+    return schema.interfaces.all();
+  });
+
+  server.get(`${INTERFACES_API}/:id`, (schema, request) => {
+    return schema.interfaces.find(request.params.id).attrs;
+  });
+
+  server.put(`${INTERFACES_API}/:id`, noop);
```

Comment:
does it make sense to remove `noop` here to get update wroking?

Reply:
fixed